### PR TITLE
Migrate ThumbnailCacheManager to User model

### PR DESCRIPTION
**Pull request by ezaquarii**
Signed-off-by: Chris Narkiewicz <hello@ezaquarii.com>

### CODE REVIEW

**Review by AndyScherzinger — APPROVED** [review]

File: src/main/java/com/owncloud/android/datamodel/ThumbnailsCacheManager.java [hunk 1]
```diff
@@ -440,7 +440,7 @@ private String getImageKey() {
 
     public static class ThumbnailGenerationTask extends AsyncTask<ThumbnailGenerationTaskObject, Void, Bitmap> {
         private final WeakReference<ImageView> mImageViewReference;
-        private static Account mAccount;
+        private User mAccount;
```
Comment (tobiasKaminsky) [inline src/main/java/com/owncloud/android/datamodel/ThumbnailsCacheManager.java:443]:
```suggestion
        private User user;
```

Then type and name matches :)

**Review by tobiasKaminsky — CHANGES_REQUESTED** [review]
Variable rename

File: src/main/java/com/owncloud/android/datamodel/ThumbnailsCacheManager.java (hunk 1 above)
Comment (ezaquarii) [reply src/main/java/com/owncloud/android/datamodel/ThumbnailsCacheManager.java]: @tobiasKaminsky done

**Review by tobiasKaminsky — APPROVED** [review]

**Review by AndyScherzinger — APPROVED** [review]

**Pull request merged**